### DISCUSSION TITLE
Banned from using GitHub Actions?

### DISCUSSION THREAD

**Discussion by infinitepower18** [community/community#25184] [Actions]
<p>Recently all my actions are failing to run and the below error is given… have I been banned from using Actions? I can’t think of a reason I may have abused the service, probably because I set them to run every 5 mins? If I’m not wrong I haven’t seen any rule not to run a workflow so frequently. Can anyone tell me what is going on?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/fb7d1d360ec858060620d3c84bfc4745db5d1ab7.png" data-download-href="/uploads/short-url/zSLTQKl1GWdrH3RQdz4lhYO1RiL.png?dl=1" title="Annotation 2020-03-23 165405.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/44692189/181089290-e51278cc-38e4-4f9a-a39b-a9fb0474c4b7.png" alt="Annotation 2020-03-23 165405.png" data-base62-sha1="zSLTQKl1GWdrH3RQdz4lhYO1RiL" width="690" height="309" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/f/fb7d1d360ec858060620d3c84bfc4745db5d1ab7_2_690x309.png, https://aws1.discourse-cdn.com/github/optimized/2X/f/fb7d1d360ec858060620d3c84bfc4745db5d1ab7_2_1035x463.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/f/fb7d1d360ec858060620d3c84bfc4745db5d1ab7_2_1380x618.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/fb7d1d360ec858060620d3c84bfc4745db5d1ab7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Annotation 2020-03-23 165405.png</span><span class="informations">1417×635 20 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/codeinteger6">@codeinteger6</a> ,</p>
<p>I have helped you report this question to the the appropriate engineering team for further investigation and evaluation. If they have any update, I will notify you in time, and sometimes the appropriate engineers may directly reply you here.</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/codeinteger6">@codeinteger6</a> ,</p>
<p>This problem is related to a recent reported incident for GitHub, this incident caused some users were mistakenly marked as abusive.</p>
<p>The engineering team has taken mitigation measures, you should be unblocked now. Please check if the problem has gone on your side. Any progress, feel free to tell us.</p>
<p>Apologize for the inconvenience caused to you.</p>

**Reply by Rahulsharma45** [reply]
> [@codeinteger6](/codeinteger6) ,
> 
> This problem is related to a recent reported incident for GitHub, this incident caused some users were mistakenly marked as abusive.
> 
> The engineering team has taken mitigation measures, you should be unblocked now. Please check if the problem has gone on your side. Any progress, feel free to tell us.
> 
> Apologize for the inconvenience caused to you.

MY ACCOUNT ACTION IS ALSO BANNED 
PLEASE FIX

**Reply by KMohZaid** [reply]
i can't use github action

**Reply by ghost** [reply]
mine too

**Reply by pradeep-lit** [reply]
I cant use tooo

**Comment by infinitepower18** [comment]
<p>They’re working fine now, thank you!</p>

**Comment by paxcoder** [comment]
my gh actions even don't send emails, nothing happens, did I got banned too?

**Reply by KMohZaid** [reply]
You will get error that GitHub action is disabled for this repo by staff...

I just open follow ticket on support page and they unban me after I removed any repository that I had fork and it was breaking GitHub action usage terms and conditions

**Comment by Munirul-haq** [comment]
I have too

**Reply by MSimsek07** [reply]
Can you use the actions now?

**Comment by MSimsek07** [comment]
Can you guys help me too? Im banned from using actions @BrightRan

**Comment by cenwadike** [comment]
Hi @BrightRan 

My github action got banned and I can't reach the support page....

How do I unban github action?
